packages/terra-table/src/subcomponents/ColumnHeaderCell.jsx

```diff
@@ -391,9 +391,15 @@ const ColumnHeaderCell = (props) => {
         className={cx('header-container')}
         {...hasButtonElement && { ref: columnHeaderCellRef, role: 'button' }}
         tabIndex={buttonTabIndex}
+        aria-sort={sortIndicator}
```

Comment:
This attribute is not fully supported by most of the screen readers.  Will this really resolve an issue?  What is not being covered by the hidden text?

Reply:
It was originally requested to add aria-sort to the column header so that it would read the sort order. However upon discovery that JAWS and most screen readers do not support this I have changed the approach to using `aria-live` instead on the sorting text.